TemporalConvolution.lua

```diff
@@ -13,7 +13,6 @@ function TemporalConvolution:__init(inputFrameSize, outputFrameSize,
     local nInputPlane = 1 -- single channel
     local nOutputPlane = outputFrameSize
     self.inputFrameSize = inputFrameSize
-    self.outputFrameSize = outputFramesize
```

Comment:
I don't see it used anywhere in this file, am I mistaken? Also there is a typo (outputFramesize instead of outputFrameSize) - so it is actually set to nil in this code.